docs/dpdk/gnmi-ctl.rst

```diff
@@ -0,0 +1,288 @@
+gnmi-ctl executable
+-------------------
+
+gnmi-ctl is a gRPC-based C++ network management interface client to handle port configurations and program fixed functions in the P4 pipeline. This client connects
+to the Config monitoring service running as part of Stratum and integrated as part of
+infrap4d. This utility is built as part of networking recipe and is available under
+`install/bin/`.
+
+gnmi client connects to gRPC port 9339 opened by the server and sends a protobuf
+based message. Refer to gnmi.proto and gnmi_ext.proto for more details on type of
```

Comment:
If you want to specify the location of `gnmi.proto` and `gnmi_ext.proto`, they're in `stratum/proto/gnmi` under the networking recipe.